app/src/main/res/values/strings.xml

```diff
@@ -940,6 +940,8 @@
     <string name="travel_empty_state_text_2" translatable="false">- Susan Sontag</string>
     <!--button for entering the search page-->
     <string name="travel_empty_state_button">Explore a city</string>
+    <!--button for entering the search page-->
+    <string name="travel_empty_state_button_new">EXPLORE A CITY</string>
```

Comment:
Don't have to add this new string. We could make the string all capital in code.